### PR TITLE
[TR] PIM-4646: Add WYSIWYG in performances guide

### PR DESCRIPTION
**Pull request by willy-ahva**

### CODE REVIEW

File: reference/performances_guide/index.rst [hunk 1]
```diff
@@ -36,3 +36,17 @@ Here is the complete formula to check if you have, by far, more indexes than the
     + N enabled locales * N enabled channels (for the completeness filters)
     + 3 for family, groups and categories
     > 64
+
+Work with WYSIWYG :
+-------------------
+
+WYSIWYG are javascript component and so are computed client side. Hardware power of the client can affect the speed of pages with WYSIWYG.
```
Comment (rybus) [inline reference/performances_guide/index.rst]: components [...] and are executed on client side. Hardware performances [...]

File: reference/performances_guide/index.rst [hunk 2]
```diff
@@ -36,3 +36,17 @@ Here is the complete formula to check if you have, by far, more indexes than the
...
+
+That's why it's not recommended to have more that one hundred WYSIWYG in the same page. In the product edit form, only attributes in the
+current attribute group and the current scope are rendered. It means, you should not have more than one hundred WYSIWYG in an attribute group
```
Comment (rybus) [inline reference/performances_guide/index.rst]: It means that you

File: reference/performances_guide/index.rst [hunk 3]
```diff
@@ -36,3 +36,17 @@ Here is the complete formula to check if you have, by far, more indexes than the
...
+and a scope. In mass edit and variant group, all scopes are rendered in the same time. It means you should not have more than one hundred
```
Comment (rybus) [inline reference/performances_guide/index.rst]: "In mass edit and variant group" > unclear, do you mean "when performing mass edit or managinf variant groups" ?

Comment (nidup) [inline reference/performances_guide/index.rst]: when configuring mass edit common attributes or editing variant group attributes

Comment (phaseinducer) [inline reference/performances_guide/index.rst]: "at the same time"

File: reference/performances_guide/index.rst [hunk 4]
```diff
@@ -36,3 +36,17 @@ Here is the complete formula to check if you have, by far, more indexes than the
     + N enabled locales * N enabled channels (for the completeness filters)
     + 3 for family, groups and categories
     > 64
+
+Work with WYSIWYG :
+-------------------
+
+WYSIWYG are javascript components and so are executed client side. Hardware performances of the client can affect the speed of pages with WYSIWYG.
```
Comment (rybus) [inline reference/performances_guide/index.rst]: executed on the

**Comment by rybus** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**